electron-builder.json

```diff
@@ -18,15 +18,22 @@
     ]
   },
   "linux": {
-    "category": "Audio",
+    "category": "AudioVideo",
     "icon": "build/icons/icon.icns",
     "target": [
       "AppImage",
       "deb",
       "flatpak",
       "pacman",
       "rpm"
-    ]
+    ],
+    "desktop": {
+      "Actions": "about;",
+      "Categories": "AudioVideo",
+      "Comment": "Advanced multi-source music streaming + discovery client",
+      "MimeType": "x-scheme-handler/muffon;",
```

Comment:
These are added to add the custom action at the end, otherwise electron-builder would add these at the end